body-tracking-samples/extern/nlohmann_json/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+include(FetchContent)
+
+FetchContent_Declare(nlohmann_json
```

Comment:
Still, please teach me how nlohmann json worked before without this change?

Reply:
We are using nuget package for VS
`<package id="nlohmann.json" version="3.7.0" targetFramework="native" />`

Similarly we are using nuget for GLFW for simple_3d_viewer.
`<package id="glfw" version="3.3.0" targetFramework="native" />`
and for cmake we have glfw as a git submodule.